docs/tutorials/python/linear-regression.md

```diff
@@ -18,8 +18,8 @@ how to encode a dataset into an iterator that MXNet can use. The data used in th
 
 ```python
 #Training data
-train_data = np.array([[1,2],[3,4],[5,6],[3,2],[7,1],[6,9]])
-train_label = np.array([5,11,17,7,9,24])
+train_data = np.random.uniform(0, 1, [100, 2])
```

Comment:
Keep in mind that target audience for these tutorials (especially LR) is beginners. This is a good opportunity to briefly mention about the importance of feature normalization.